extensions/sql-migration/package.json

```diff
@@ -178,7 +178,8 @@
     "@microsoft/ads-service-downloader": "^1.2.1",
     "@microsoft/ads-extension-telemetry": "^3.0.1",
     "uuid": "^8.3.2",
-    "vscode-nls": "^4.1.2"
+    "vscode-nls": "^4.1.2",
+    "@azure/storage-blob": "^12.23.0-beta.1"
```

Comment:
You need to run `yarn` (or `yarn install`) and then commit the updated lockfille as well

Reply:
done